backend/packages/Upgrade/src/api/controllers/QueryController.ts

```diff
@@ -50,4 +50,40 @@ export class QueryController {
   ): Promise<any> {
     return this.queryService.analyze(dataLogParams.queryIds, request.logger);
   }
+
```

Comment:
Why do we need a separate http endpoint for archive? I thing We should use the /analyse endpoint and server should get the data from the archive table in case the experiment is in archive state. Given that we have the same response type we can use the same endpoint. This will also keep the frontend dumb.

Reply:
code updated